client/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -1,30 +1,38 @@
-import React, { useState } from 'react'
-import { Link } from 'react-router-dom'
+import React, { useEffect, useState } from 'react'
+import { useHistory, Link } from 'react-router-dom'
 
+import { useAppDispatch, useAppSelector } from '@client/store'
+import { LoginActions } from '@client/store/login'
 import { Urls } from '@client/utils'
 import { useTranslation } from 'react-i18next'
 import { Objects } from '@core/utils'
 import { BasePaths } from '@client/basePaths'
-import { ApiEndPoint } from '@common/api/endpoint'
-import { useAppDispatch } from '@client/store'
-import { LoginActions } from '@client/store/login'
 import { LoginValidator } from '@client/pages/Login/utils/LoginValidator'
 
 const LoginForm: React.FC = () => {
-  const { i18n } = useTranslation()
   const dispatch = useAppDispatch()
-  const invitation = Urls.getRequestParam('invitation')
-  const isInvitation = !Objects.isEmpty(invitation)
+  const { i18n } = useTranslation()
+  const history = useHistory()
+
+  const invitationUuid = Urls.getRequestParam('invitationUuid')
+  const isInvitation = !Objects.isEmpty(invitationUuid)
+  const { user: invitedUser } = useAppSelector((state) => state.login.invitation)
 
   const [email, setEmail] = useState<string>('')
   const [password, setPassword] = useState<string>('')
-  const [errors, setErrors] = useState<Record<string, string>>({})
-
-  // TODO:
-  // Handle invitation (=> get invited user details (email))
   const [password2, setPassword2] = useState<string>('')
 
+  const [errors, setErrors] = useState<Record<string, string>>({})
+  useEffect(() => {
+    dispatch(LoginActions.resetLogin())
+    dispatch(LoginActions.initLogin())
```

Comment:
this doesn't make much sense.
better to dispatch reset login state within initLogin action